source/ide/ide_methods.bas

```diff
@@ -17744,8 +17734,7 @@ SUB Help_ShowText
             Help_LineLen(y - Help_sy) = x3 - 1
 
             FOR x4 = 1 TO Help_wx2 - POS(0) + 1
-                IF col = 0 THEN col = 7
-                COLOR col AND 15, col \ 16
+                COLOR 7, (col - 128) \ 16

```

Comment:
Do you not want to set the foreground color with `col AND 15`?

Reply:
Not in that place. It wouldn't make a visible difference, but would need more processing time. That loop basically just fills the empty space from help line end to window width with the current bg color, the fg color is completely irrelevant here.